src/cmd/run.go

```diff
@@ -57,6 +69,41 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := "opslevel-runner-leader-lock"
```

Comment:
if we run multiple opslevel-runners in a k8s cluster this will be share'd amongst them and then scaling is broken.  This needs to be unique based on the deployment.